apps/emqx_management/src/emqx_mgmt_api_listeners.erl

```diff
@@ -642,7 +644,7 @@ create(Path, Conf) ->
     wrap(emqx_conf:update(Path, {create, Conf}, ?OPTS(cluster))).
 
 ensure_remove(Path) ->
-    wrap(emqx_conf:remove(Path, ?OPTS(cluster))).
+    wrap(emqx_conf:update(Path, emqx_schema:tombstone(), ?OPTS(cluster))).
```

Comment:
Nit: Maybe add `emqx_conf:tombstone(KeyPath, Opts)` (since we have e.g. emqx_conf:remove/2)

Reply:
that'll introduce a new bpapi ?

Reply:
sorry, i forgot it's already added, just forgot to call it.